src/common/api/entities/tutanota/TypeRefs.ts

```diff
@@ -1303,8 +1363,8 @@ export type MailFolder = {
 	_ownerGroup: null | Id;
 	_ownerKeyVersion: null | NumberString;
 	_permissions: Id;
+	color: null | string;
 	folderType: NumberString;
-	isLabel: boolean;
```

Comment:
Wouldn't it be better to keep the `isLabel` flag instead of adding a new `MailSetKind.LABEL`?

We initially came up with this model in order to allow a LABEL with the `MailSetKind.All`, that allows to filter for/display all mails in the mailbox with a single click, but this `All` MailSet does not necessary need to be a folder. This `All` MailSet furthermore enables to simplify the search indexing of mails (e.g. load all mails for a certain date range).